.github/workflows/merged.yml

```diff
@@ -179,7 +179,6 @@ jobs:
           bundle install
 
           bundle exec fastlane testWithoutCoverageForUITests scheme:"OneLoginBuild" \
-            configuration:"Build" \
```

Comment:
Assume this is needed to run the tests in the Debug mode?

Reply:
Yep 🙂